curriculum/structure/superblocks/relational-databases-v9.json

```diff
@@ -3,8 +3,89 @@
     {
       "dashedName": "relational-databases",
       "modules": [
+        {
+          "dashedName": "code-editors",
+          "blocks": ["lecture-working-with-code-editors-and-ides"]
+        },
+        {
+          "dashedName": "bash-fundamentals",
+          "blocks": [
+            "lecture-understanding-the-command-line-and-working-with-bash",
+            "workshop-bash-boilerplate",
+            "review-bash-commands",
+            "quiz-bash-commands"
+          ]
+        },
+        {
+          "dashedName": "sql-and-postgresql",
+          "blocks": [
+            "lecture-working-with-relational-databases",
+            "workshop-database-of-video-game-characters",
+            "review-sql-and-postgresql",
+            "quiz-sql-and-postgresql"
+          ]
+        },
+        {
+          "dashedName": "lab-celestial-bodies-database",
+          "blocks": ["lab-celestial-bodies-database"]
```

Comment:
I made the labs/cert projects standalone modules but kept the exam marked as coming soon, since we agreed to release the first two certs first.

Not sure if this might cause confusion. I can revert it to match FSD if that's clearer for now.